tests/QMLEngine/basic.js

```diff
@@ -29,4 +29,9 @@ describe('QMLEngine.basic', function() {
     expect(qml.inner3).toBe(qml.current + 'foo/foo/lol/');
     expect(qml.full).toBe('http://example.com/bar');
   });
+
+  it('the signal disconnect in the signal handler with many subscribers', function() {
```

Comment:
Could this have a bit shorter name with this explanation (i.e. what it does) placed into a comment inside the qml file?